src/status_im/ui/screens/events.cljs

```diff
@@ -252,12 +259,10 @@
    (let [{:transport/keys [chats]} db
          sharing-usage-data? (get-in db [:account/account :sharing-usage-data?])]
      (handlers-macro/merge-fx cofx
-                              {:dispatch-n (concat [[:initialize-db encryption-key]
-                                                    [:load-accounts]
-                                                    [:listen-to-network-status]
-                                                    [:navigate-to :accounts]]
-                                                   (when sharing-usage-data?
-                                                     [[:unregister-mixpanel-tracking]]))}
```

Comment:
@jeluard I know you worked on mixpanel, this event does not seem to be used anymore, do you have any context on it or it's just dead code?

Reply:
It was all done locally apparently I forgot to force push sorry for that